src/main/java/com/careerfair/q/service/validation/implementation/ValidationServiceImpl.java

```diff
@@ -62,4 +63,35 @@ public void checkEmployeeAssociations(String companyId, String employeeId, Role
             throw new ValidationException(ex.getMessage());
         }
     }
+
+    @Override
+    public <T extends StudentRequest> void checkValidStudentRequest(T studentRequest)
+            throws ValidationException {
+        if (studentRequest.firstName == null || studentRequest.firstName.isEmpty()) {
```

Comment:
Have a method and pass it in there to throw exception

Reply:
ok